### PR TITLE
[ICP-11256] Everspring Floodlight - removed Illuminance Measurement capability 

### PR DESCRIPTION
**Pull request by PKacprowiczS**
@greens @dkirker @KKlimczukS

### CODE REVIEW

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy [hunk 1]
```diff
@@ -1,5 +1,5 @@
 /**
- *  Copyright 2018 SmartThings
```
Comment (tpmanley) [inline devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:2]: The copyright date should not be changed

Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:2]: Actually that's a 'fix', I've created this DTH last year, but apparently copied copyright statement without noticing its old date.

Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:2]: @tpmanley Soooo, should I change it back to 2018?

Comment (tpmanley) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:2]: You can leave it

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy [hunk 2]
```diff
@@ -16,7 +16,6 @@ metadata {
 	definition(name: "Z-Wave Motion Light", namespace: "smartthings", author: "SmartThings", ocfDeviceType: "oic.d.light", mmnm: "SmartThings", vid: "generic-motion-light") {
```
Comment (greens) [inline devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:16]: this metadata still has `illuminance`

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy [hunk 3]
```diff
@@ -68,15 +56,10 @@ def updated() {
 }
 
 def configure() {
-	def cmds =  [
-			secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
-			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03)),
-			secure(zwave.switchBinaryV1.switchBinaryGet())
+    [
```
Comment (greens) [inline devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy]: nit: spacing

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy (hunk 2 above)
Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:16]: No, it hasn't. I've removed it in MR 1017

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy [hunk 4]
```diff
@@ -68,15 +56,10 @@ def updated() {
 }
 
 def configure() {
-	def cmds =  [
-			secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
-			secure(zwave.sensorMultilevelV5.sensorMultilevelGet(sensorType: 0x03)),
-			secure(zwave.switchBinaryV1.switchBinaryGet())
+	[
+        secure(zwave.notificationV3.notificationGet(notificationType: 0x07)),
```
Comment (greens) [inline devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy]: you sure you don't have a spaces->tabs function in your editor?

Comment (PKacprowiczS) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy]: I've recently switched to VS Code, and I was sure I've changed it. Anyway, there should be no more spacing issues from now.

**Review by dkirker — APPROVED** [review]

File: devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy (hunk 2 above)
Comment (dkirker) [reply devicetypes/smartthings/zwave-motion-light.src/zwave-motion-light.groovy:16]: @PKacprowiczS You will need to request the metadata to production in the portal (I can't do that as of now, unfortunately). That will publish it in to acceptance. Technically we are supposed to only publish to acceptance on Tuesdays. I'll go ahead and merge this PR. I suppose that we can just request the metadata to acceptance (or, just do that next Tuesday which will line up with this PR getting deployed to staging and acceptance) before hand. I'll deploy the metadata to production when this is deployed to production.

**Pull request merged**